packages/twenty-front/src/pages/settings/data-model/SettingsObjectFieldEdit.tsx

```diff
@@ -172,6 +172,10 @@ export const SettingsObjectFieldEdit = () => {
     navigate(`/settings/objects/${objectSlug}`);
   };
 
+  const shouldDisplaySaveAndCancel =
```

Comment:
@thomtrp I rather create an utils here, as we're going to add type in this like `MultiSelect` and maybe other ones if needed

Reply:
@magrinj I updated by creating a separated function for that. I am not a big fan of creating utils file when the function is used only in one place. We'll see if we need to move it at some point. And for now adding a new type will simply be a new condition in that function.